### PR TITLE
sf2 plugin redesign

### LINKED ISSUE #6454: Bad selection of colors on SF2 UI
### Bug Summary
Visual impaired can see the settings for BANK & PATCH.
Dark-purple on black is not functional

#### Steps to reproduce
Just look at it

#### Expected behavior
Clear and distinctable  UI-text

#### Actual behavior
Not distinctable  UI-text

#### Screenshot
image

#### Affected LMMS versions
< 1.2. 

#### Type of Issue
Good fist Issue

### PR DESCRIPTION
**Pull request by Rossmaxx**
I have edited the colours of patch and bank lcd num boxes to make it more visible. fixes #6454. Also includes some other minor changes.

this is how it looks : 
image

note that when a file is selected, the file selected icon turns dark green.

### CODE REVIEW

**Comment by sakertooth** [comment]
You can add special keyword syntax that automatically closes the issue pertaining to this PR.
Add something along the lines of "Fixes hashtag 6469" in the description of your PR (I didn't want to reference the issue here).
More info: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

Not mandatory, but helps.

**Comment by Rossmaxx** [comment]
for some reason, the logo seems to be broken.

**Comment by qnebra** [comment]
I am strongly against this artwork. Ugly, amateurish, not fitting into overall style of lmms, make it feel like some amateur crap.

**Comment by Rossmaxx** [comment]
> Ugly, amateurish, not fitting into overall style of lmms

I agree. This is not the final artwork. This is still a draft. I am collecting ideas for the final version by using this artwork as a test. Are the colors ok or should i change em.

**Comment by zonkmachine** [comment]
The 21pink lcd was added nine years ago with sf2player but in 1.2.2 somehow it's the green lcd so it seem to have been failing the pink one earlier and falling back to some default. The loading of the right lcd then was corrected and we ended up with this one that turned out to be unreadable. @Rossmaxx Either change back the lcd to the earlier one, like you do here, or brighten up the 21pink one (it's in the themes). sf2player seem to be the only instance using this graphic so it should be safe modifying it. I think it's better to start with a small bite if you haven't submitted a PR before so my advice is to just fix the lcd and skip the rest for now.

**Comment by Veratil** [comment]
Pull the changes for 21pink to 19green and open another PR to only fix #6454 please. :)

File: data/themes/default/style.css [hunk 1]
```diff
@@ -858,7 +858,7 @@ lmms--gui--OrganicInstrumentView lmms--gui--Knob#volKnob {
 }
 
 lmms--gui--Sf2InstrumentView lmms--gui--Knob {
-	color: #ff00ea;
+	color: #00ff6e;
```
Comment (zonkmachine) [inline data/themes/default/style.css:861]: What are these changes for?

Comment (Rossmaxx) [reply data/themes/default/style.css:861]: To get green colour on the knob. I have edited some pngs too to get green colour on areas where there was pink before

Comment (zonkmachine) [reply data/themes/default/style.css:861]: Aight!

**Review by zonkmachine — APPROVED** [review]

**Comment by zonkmachine** [comment]
It would be nice to see a clean PR though. In this case I meań rebasing and removing the commits that touch things which are not to be changed.

**Comment by Rossmaxx** [comment]
closing in favour of a new PR

**Pull request closed without merging**